requirements.txt

```diff
@@ -1,17 +1,18 @@
 py-solc
-web3
-eth-tester[py-evm]
+web3[tester]>=5.0.0b2
```

Comment:
We should not install the tester dependencies, because we dont use them.

Reply:
`web3[tester]>=5.0.0b2`

Reply:
the master branch already has `eth-tester[pyevm]` as dependency. The plugin imports 'eth_tester'. so, AFAICT we do use the tester dependencies (at least some of them).

Also web3 pins those dependencies, but AFAICT only when you specify the extra requirement. I'd like to have this pinning when running a simple 'pip install contract-deploy-tools'.


I did add those extra dependencies because I ran into problems (and only when I tried to use this package in the blockchain repo).

Reply:
I can make the suggested changes, but we do lose the pinning that web3 gives us for free. 

If we upgrade web3 in the future, we have to make sure to also upgrade eth-tester to the right version. `pip install contract-deploy-tools` will not get the right version of eth-tester automatically.

Reply:
if we need the right version, than we should maybe pin itin our setup.py file?

Reply:
but then we need to pin web3 too, that's too restrictive.